packages/flutter/lib/src/material/material.dart

```diff
@@ -100,6 +101,14 @@ class Material extends StatefulComponent {
   }
 
   _MaterialState createState() => new _MaterialState();
+
+  void debugFillDescription(List<String> description) {
```

Comment:
We seem to have two names for this sort of function (the other is debugDescribeSettings).  Should we unify on one name?